### ISSUE TITLE
Better throttle feedback/locking

### ISSUE THREAD

**Issue by InsaneGrox** [boyestrous/elite-vr-cockpit#20]
Currently the throttle movement is, well, hard to control, complete stopping is impossible without a reset throttle to 0 holo button, I've got a few potential solutions (which should probably be optional and not forced upon players)
1: have a deadzone near the center where you can snap the throttle at 0, in addition, have a haptic pulse when entering/exiting this deadzone (this I think would work the best in most situations, also the original elite VR cockpit actually did the first part of this)
2: have the throttle reset to 0 upon being let go, this would not be ideal for long trips like hutton orbital but this would be a solution for stopping.
3: changing the throttle from position based to rotation based, the thrustmaster hotas X throttle is basically an analog lever mechanically speaking, and rotation is easier to remember where you are, would also be less susceptible to tracking issues.

**Comment by boyestrous** [comment]
You're absolutely right. I've gotten used to using the Throttle Zero holo button, but it's not ideal. 

I really like the idea of the deadzone + haptic pulse. I had a working version at one point, but I had to scrap it because it interfered with a different (more important) change I needed to make. I could probably bring it back relatively quickly.

I was also considering a detent style lock that could be toggled by the user, so the throttle would stop at zero and not proceed into reverse until the player pressed a button. 

Options 2 and 3 are bigger changes and I can't quite see that they offer tangible advantages over Option 1, so that seems like the logical place to go in the near term. I do see the possibility of adding them as options that could be configured by the player, if they wanted it that way.

I'm going to take a look this weekend and see if I can re-implement the haptic feedback/deadzone. Maybe I can roll it into the release with the OnFoot controls.

**Comment by InsaneGrox** [comment]
Gonna just give context for what I mean by rotation based throttle by showing the physical equivilant for anyone who's wondering what I mean

https://github.com/boyestrous/elite-vr-cockpit/assets/54233743/2174f835-ff4b-4a25-9f56-a08d0d6da152
